platform/docker/docker-compose.yml

```diff
@@ -10,4 +10,14 @@ services:
       - "10001:10001"
       - "10002:10002"
     volumes:
-      - "./azurite/data:/data"
\ No newline at end of file
+      - "./azurite/data:/data"
+  sqldata:
+    image: mcr.microsoft.com/mssql/server:2017-latest
```

Comment:
Any particular reason why the `2017-latest` image was chosen over the [other options](https://mcr.microsoft.com/v2/mssql/server/tags/list)?

Reply:
Pushed changes